devicemodel/hw/vdisplay_sdl.c

```diff
@@ -611,6 +612,22 @@ sdl_gl_display_init(void)
 	return;
 }
 
+static void sdl_gl_prepare_draw(struct vscreen *vscr)
+{
+	SDL_Rect bogus_rect;
+
+	if (vscr == NULL)
+		return;
+
+	bogus_rect.x = 0;
+	bogus_rect.y = 0;
+	bogus_rect.w = 32;
+	bogus_rect.h = 32;
+	SDL_RenderClear(vscr->renderer);
+	SDL_RenderCopy(vscr->renderer, vscr->bogus_tex, NULL, &bogus_rect);
```

Comment:
Please add some comments here or a TODO for monitoring SDL update.

Reply:
> Please add some comments here or a TODO for monitoring SDL update.
It is updated in the new version.